lua/telekasten.lua

```diff
@@ -245,36 +255,38 @@ local function make_config_path_absolute(path)
 end
 
 local function recursive_substitution(dir, old, new)
-    if not global_dir_check() then
-        return
-    end
+    global_dir_check(function(dir_check)
```

Comment:
so most changes look like this, code is the same but now it includes a `function` callback